ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -1345,3 +1378,27 @@ static void ReturnFileMutexIterator(FileMutexMap::iterator lockIterator)
         s_fileLocks.erase(lockIterator);
     }
 }
+
+static const char* GetRelativePath(const char* path, const char* root)
+{
+    size_t rootLength = strlen(root);
+    size_t pathLength = strlen(path);
+    if (pathLength < rootLength || 0 != memcmp(path, root, rootLength))
+    {
+        fprintf(stderr, "GetRelativePath: root path '%s' is not a prefix of path '%s'\n", root, path);
```

Comment:
This function is another spot we could use a `// TODO(Mac): Add this message to PrjFSLib logging once available (#395)`